.github/workflows/cabal.project.default.local

```diff
@@ -1,5 +1,8 @@
 package cardano-crypto-praos
   flags: -external-libsodium-vrf
 
+package strict-checked-vars
+  flags: +checktvarinvariants +checkmvarinvariants
```

Comment:
I think we can move this to [`asserts.cabal`](https://github.com/IntersectMBO/ouroboros-consensus/blob/6b130b1eb11db3b367c7cc3cd3bb2bb4496d6b8a/asserts.cabal).

This file is only used by the GitHub Actions CI; while `asserts.cabal` is also used on Hydra and when building/testing locally.